src/external/rawspeed/data/cameras.xml

```diff
@@ -5734,6 +5734,40 @@
 		<Crop x="0" y="0" width="-74" height="0"/>
 		<Sensor black="15" white="3971"/>
 	</Camera>
+	<Camera make="Panasonic" model="DMC-LX15">
+		<ID make="Panasonic" model="DMC-LX15">Panasonic DMC-LX15</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">GREEN</Color>
+			<Color x="1" y="0">BLUE</Color>
+			<Color x="0" y="1">RED</Color>
+			<Color x="1" y="1">GREEN</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
+		<Sensor black="142" white="4095" iso_list="80 100 125 400"/>
```

Comment:
> For one file it should return the same value as dngmeta.sh, or?

Yes, i expect that when called for a single DNG, it should provide the same output as `dngmeta.sh`

BTW: i plan on eventually dropping `dngmeta.sh`, once `dngmeta.rb` fully supersedes it. So if `dngmeta.rb` has any bugs, better raise them in redmine.